pkg/volume/secret/secret_test.go

```diff
@@ -124,8 +123,7 @@ func TestPlugin(t *testing.T) {
 	doTestSecretDataInVolume(volumePath, secret, t)
 
 	metrics, err := builder.GetMetrics()
-	assert.NotEmpty(t, metrics)
-	assert.NoError(t, err)
+	volume.AssertMetricsOk(t, metrics, err)
```

Comment:
Maybe add a comment "beware those who dare venture here..."? I'm a firm believer that unit tests should be small and targeted--test one thing--so that if it is skipped or it fails, it's obvious what happened/broke. But, we often end up with one unit test that tests n different things. So, I'll let you decide what the best approach is.

Didn't mean to drag this code review out with nit picky stuff!